Google.PowerShell.IntegrationTests/Logging/Logging.GcLogSink.Tests.ps1

```diff
@@ -70,7 +70,68 @@ Describe "Get-GcpsLogSink" {
     }
 }
 
-Describe "New-GcpsLogSink" {
+function Test-GcLogSink (
+    [string]$name,
```

Comment:
Can't you do the asserts in this function? That would give better test failure messages